javascript/javascript2/week2/main.css

```
@@ -0,0 +1,84 @@
+body {
+    font-family: "Poppins", sans-serif;
+    /*background-color: #F9FBFD;*/
+    background-image: 
+    linear-gradient(to right, rgba(38, 49, 8, 0.52), rgba(240, 189, 20, 0.73)),
+    url("images/clock.jpg");
+background-position:0vh;
+color:white;
```

Consistent indentation.
```suggestion
    background-position:0vh;
    color:white;
```

---

will be careful the next time